org.gnome.Eolie.json

```diff
@@ -0,0 +1,314 @@
+{
+  "app-id": "org.gnome.Eolie",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "eolie",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=host",
```

Comment:
Well you have two options:

Best option: Use the file chooser to allow the user to import profiles, this leaves the directories not readable unless the user explicitly allows it.

Acceptable option: Just grant read-only access to those two specific directories.